src/Compiler/CompilerUtils.cpp

```diff
@@ -162,6 +169,23 @@ int Command::exec(std::string wdir) const {
   return 0;
 }
 
+void showCompilePhase(std::string msg) {
+  time_t rawtime;
+  struct tm *timeinfo;
+  char buffer[80];
+
+  // Get current date.
+  time(&rawtime);
+  timeinfo = localtime(&rawtime);
+  strftime(buffer, 80, "%c", timeinfo);
+  std::string currentTime(buffer);
+
+  // Use blue bold text.
+  llvm::outs() << "[\033[34;1m" << CURRENT_COMPILE_PHASE++ << "/"
```

Comment:
ANSI escape sequence isn't support on all terminal types. And different terminal might even interpret the sequence differently. So unless the color adds a lot of value, I would just leave it out.

Reply:
Ok. Removed it. Thanks!